nano/node/election.hpp

```diff
@@ -79,6 +79,8 @@ class election final : public std::enable_shared_from_this<nano::election>
 	bool stopped;
 	std::unordered_map<nano::block_hash, nano::uint128_t> last_tally;
 	unsigned confirmation_request_count{ 0 };
+	std::chrono::steady_clock::time_point last_broadcast {};
+	std::chrono::steady_clock::time_point last_request {};
```

Comment:
Does value initializing `{}` these have a benefit?

Reply:
As you noted, default constructor does the same (setting to zero)!